### ISSUE TITLE
SM: Potent Enchantments

### ISSUE THREAD

**Issue by lastadm** [WarEmu/WarBugs#8167]
Tactic broken.

1. Damage varies depending on enchant on blade. (Very strange xD)
2. Not physical damage. (See mitigations from AA)
3. Tactic can be critical hit.

![image](https://cloud.githubusercontent.com/assets/18344297/19767246/4d709b8e-9c6b-11e6-8fa4-fc06fc114b97.png)

**Comment by Luth-RoR** [comment]
Refering to #8139

**Comment by ozarmidrashim** [comment]
Point 2: https://taugrim.com/2009/08/16/war-swordmaster-guide-mechanics-part-2/ says it was Spirit and this fits with logic of tactic.

**Comment by lastadm** [comment]
When DMG is not Physical, we can see it in tooltips.

**Comment by Luth-RoR** [comment]
No, tooltips are sometimes wrong; i once tested on the mythic server if the kotbs aura _all out assault_ can trigger procs on the enemy and found out that it did no physical damage (like the tooltip says) but elemental damage.
Another example: some debuffs have the type _cripple_ in the tooltip, which didn't exist (only curse, hex, ailment and damaging).
